.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,59 @@
+name: Build and Deploy Docs
```

Comment:
We've agreed that this workflow would be copied in the v2 branch and would be slightly modified in order to fit its needs